contributor/video-proposal.md

```diff
@@ -18,47 +18,50 @@ This file represents the text intended for the video proposal (as-described in [
 ## Description & Learning Promise
 
 ### What is this video about, and why is it important?
-_Write a paragraph that answers this question._
+The Contributor role is one the most central roles in InnerSource - without Contributors, there would not be anything to receive or discuss about. 
+There's far more to it than just dumping some stuff in a branch or issue.
+This segment teaches allows you to learn about the mindset, the habits, technical approaches and other details around this role enabling you to successfully contribute to the host teams work, foster good relationships with them and hopefully be a happy InnerSource contributor.
 
 ### What you’ll learn—and how you can apply it
-_Provide a few bullet points to answer this question._
 
-* 
-* 
-*
+* Introduction: How to contribute to an InnerSource project in a good and what it means to be an InnerSource Contributor  
+* Becoming an InnerSource Contributor: Mindsets and habits of successful contributors
+* The Contributor Ethos: Getting more specific on execution of the mindset and habits   
+* Mechanics of Contributing: More technical aspects of contribution to ensure success
+* Benefits of Contribution: What's in it for me (the guest), the receiving team (the host) and our company?
+* Conclusion: Recap of this section and pointers to more content
 
 ### This video is for you because…
-_Think about roles to goals here._
 
-* 
-* 
-* 
+* You are a possible future Contributor to an InnerSource project and want to know what to expect 
+* You are a host team member looking to open your work for outside contributors / inner sourcing and want to be prepared to know what being a contributor may be like 
+* You are a product owner looking to support your TC or a (line) manager and want to know how the inner source approach is meant to work.
 
 ### Prerequisites
  
-* 
-* 
-* 
+* Please watch the [Intro to InnerSource Video](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504)
+* Basic Understanding of the Trusted Committer role
```

Comment:
Is there a specific video to watch here, perhaps?

Reply:
[Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925)?

Reply:
```suggestion
* Also watch the [Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925) to the Trusted Committer role.
```